Makefile

```diff
@@ -215,7 +215,8 @@ tmp/$(HOST)-$(PORT).key tmp/$(HOST)-$(PORT).crt: ## Self-signed cert for local H
 		-sha256 \
 		-nodes \
 		-days 365 \
-		-subj "/C=US/ST=District of Columbia/L=Washington/O=GSA/OU=Login.gov/CN=$(HOST):$(PORT)"  \
+		-subj "/C=US/ST=District of Columbia/L=Washington/O=GSA/OU=Login.gov/CN=$(HOST)"  \
+		-addext "subjectAltName=IP:$(HOST)" \
```

Comment:
If someone has existing files, it'd be nice if we could somehow invalidate those without them having to manually remove them. Maybe the output filenames should exclude the port, which has a dual effect of reflecting these changes and forcing a rebuild of existing certificates?

Reply:
Yeah I was thinking about that as well. I'll make the update--the cert should work for any port.